### DISCUSSION TITLE
Development may be paused

### DISCUSSION THREAD

**Discussion by vincentneo** [vincentneo/LosslessSwitcher#27] [Announcements]
Aside from the fact that I am often busy with other things, my amplifier (Denon PMA-50, which I use for listening as well as to test this app) has failed today, leaving me with no viable test device.

Sorry about that.

**Comment by blademan007** [comment]
Where are you located? I may have an extra amp or 2 that you could use.

**Reply by vincentneo** [reply]
Thanks but it's alright! I will be getting a new one soon!

**Reply by blademan007** [reply]
Let me know. I've multiple older AV receivers (Pioneer with full pre-outs) just sitting around gathering dust, and I'd donate one to you for your nice work.
BTW, production release works on MacBook Pro (Retina, 13-inch, Early 2015), 2.9 GHz Dual-Core Intel Core i5, MacOS 11.6.2.

**Comment by taltalon** [comment]
Fantastic work Vincent - 1.1 Build 8 works well with Local Files too! - the stable non beta release did not work for me. I'm on MacOS 13 (Ventura with lots of DACs here). How can we pay for your work? I will contribute if possible. I was looking for such an app for a LONG TIME.

**Reply by Justinwang677** [reply]
How did you get local files to work is it only in apple music or does it work with QuickTime or something else?

**Reply by vincentneo** [reply]
@taltalon I saw that you figured how to contribute already! Thanks for the contribution!

**Reply by vincentneo** [reply]
@Justinwang677 I remember that if you open a audio file using the Music app, it would be added to the library, just like how iTunes would have worked in the past.

**Comment by ashafai** [comment]
How do I get to build 8? I am currently on build 7?

**Reply by vincentneo** [reply]
Hi @ashafai, you can get build 8 here: https://github.com/vincentneo/LosslessSwitcher/releases/tag/1.1.0-beta5

To get the latest (either beta or 'stable') release, this page: https://github.com/vincentneo/LosslessSwitcher/releases will always have the latest build on the top.

**Comment by taltalon** [comment]
I got the latest beta From the main page of the project. There is a link.

**Comment by taltalon** [comment]
i just installed the latest LosslessSwitcher 1.1 Build 8 from this link
https://github.com/vincentneo/LosslessSwitcher/releases/tag/1.1.0-beta5

after installing tt says LosslessSwitcher 1.1 Build 8 - i did nothing
special.  *local files in Apple Music library* switch Sample rate without
an issue with at least 3 DACs.  Works fine also from Apple Music titles
which are not local to my machine.
